scripts folder

retokromer opened the pull request:
- create a `script` folder
- modify path accordingly

retokromer committed:
script folder

retokromer committed:
script folder

retokromer committed:
update date

retokromer committed:
script folder

retokromer committed:
correct link

kfrn commented:
I would probably have gone with "script**s**", but no big deal. LGTM, thanks @retokromer !

retokromer committed:
[WIP] update readme

- add maintainers
- add link to PR
- move licence at the end

Remarks:
- It needs more work by an English native.
- The section `AVHack Team` could be called `History` and start with Ashley’s first release.

retokromer commented:
Thank you, @kfrn!

I just did also some changes to the readme, but in my opinion needs some more refactoring.

retokromer committed:
plural

retokromer committed:
plural

retokromer committed:
plural

retokromer committed:
more coherent output prompt

retokromer committed:
more coherent output prompt
